monarchmoney/monarchmoney.py

```diff
@@ -2395,6 +2377,95 @@ async def upload_account_balance_history(
             if resp.status != 200:
                 raise RequestFailedException(f"HTTP Code {resp.status}: {resp.reason}")
 
+    async def get_recurring_transactions(
+        self, start_date: str, end_date: str
```

Comment:
I think these start / end dates need to be `Optional[str] = None`, similar to what's done in other methods. I am receiving an error with the no-args call:

```
In [6]: result = asyncio.run(mm.get_recurring_transactions())
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[6], line 1
----> 1 result = asyncio.run(mm.get_recurring_transactions())

TypeError: MonarchMoney.get_recurring_transactions() missing 2 required positional arguments: 'start_date' and 'end_date'
```

Reply:
ah, good point, fixing!